hive/src/main/java/com/netease/arctic/hive/utils/HiveMetaSynchronizer.java

```diff
@@ -95,6 +95,14 @@ private static boolean updateStructSchema(TableIdentifier tableIdentifier, Updat
     for (int i = 0; i < hiveStruct.fields().size(); i++) {
       Types.NestedField hiveField = hiveStruct.fields().get(i);
       Types.NestedField icebergField = icebergStruct.field(hiveField.name());
+      // to check if lowercase matches
+      List<Types.NestedField> fields = icebergStruct
```

Comment:
I suggest replacing the process of getting iceberg fields here with traversing all Hive fields, and checking that the matched result is no more than 1.